docs/en/ufs/Ozone.md

```diff
@@ -30,7 +30,9 @@ To configure Alluxio to use Ozone as under storage, you will need to modify the
 $ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
 ```
 
-Edit `conf/alluxio-site.properties` file to set the under storage address to the Ozone bucket and the Ozone directory you want to mount to Alluxio. 
+Edit `conf/alluxio-site.properties` file to set the under storage address to the Ozone bucket and the Ozone directory you want to mount to Alluxio.
+Ozone supports two different schemas `o3fs` and `ofs`
```

Comment:
What are the differences between `o3fs` and `ofs`? Is `ofs` currently supported in Alluxio?

Reply:
the `ozone underfs integration` suport `ofs` schema.